rviz_rendering/test/rviz_rendering/objects/point_cloud_test.cpp

```diff
@@ -281,15 +276,10 @@ TEST_F(PointCloudTestFixture, addPoints_adds_new_renderable_whenever_it_is_calle
 
 
 TEST_F(PointCloudTestFixture, addPoints_adds_vertices_with_correct_geometry_when_called) {
-  int glsl_version = testing_environment_->getGlslVersion();
   auto point_cloud = std::make_shared<rviz_rendering::PointCloud>();
   point_cloud->setRenderMode(rviz_rendering::PointCloud::RM_FLAT_SQUARES);
   size_t number_of_vertices_per_flat_square = {0};
-  if (glsl_version >= 150) {
-    number_of_vertices_per_flat_square = 1;
-  } else if (glsl_version >= 120) {
-    number_of_vertices_per_flat_square = 3 * 2;  // two triangles for one square
-  }
+  number_of_vertices_per_flat_square = point_cloud->getVerticesPerPoint();
```

Comment:
Same comment here about combining this statement with the one above.

Reply:
Addressed with 5b71a093f6beaf74f5151bdd2974995be6c8cc33